ses/Package.md

```diff
@@ -0,0 +1,20 @@
+Connects to AWS SES from Ballerina
+
+## Package overview
+
+The `ballerinax/aws.ses` is a [Ballerina](https://ballerina.io/) connector for AWS SES. It is comprised of the following capabilities.
+* Perform AWS SES related operations programmatically. The `ballerinax/aws.ses` module provides this capability.
+
+### Compatibility
+|                    | Version         |  
+|--------------------|-----------------|
+| Ballerina Language | Swan Lake Beta3 |
+| Amazon SES API     | v2              |
+
+## Report issues
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina AWS SES repository](https://github.com/ballerina-platform/module-ballerinax-aws.ses)
```

Comment:
Let's add the following text here

To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina Extended Library repository](https://github.com/ballerina-platform/ballerina-extended-library)